src/main/java/Tongdayso.java

```diff
@@ -0,0 +1,14 @@
+
+public class Tongdayso {
+
+    public int tdso(int n) {
+        int tong = 0;
+        int s1 = 1;
```

Comment:
Biến `s1` phải được khai báo ngay ở vòng `for`

Reply:
done